### PR TITLE
Fix hiding curl 404 errors outside debug mode

### PR DESCRIPTION
**Pull request by vojtechtrefny**
New version of https://github.com/vojtechtrefny/anaconda/commit/2b542027b1f4c01e77f2e9ea40e70519b0f10cd9
Previous version actually worked only because of bug in dracut and that bug is already fixed.

### CODE REVIEW

File: dracut/anaconda-lib.sh [hunk 1]
```diff
@@ -181,6 +182,13 @@ tell_user() {
     fi
 }
 
+# print something only in if debug/inst.debug/rd.debug
+debug_msg() {
+   if getargbool 0 rd.debug || getargbool 0 debug || getargbool 0 inst.debug; then
+     echo $* >&2
+  fi
```
Comment (vpodzime) [inline dracut/anaconda-lib.sh]: bad indentation of ``fi``

**Comment by vpodzime** [comment]
Other than the nitpick above this looks good to me.

File: dracut/anaconda-lib.sh (hunk 1 above)
Comment (vojtechtrefny) [inline dracut/anaconda-lib.sh]: Thanks, fixed locally.

**Comment by vojtechtrefny** [comment]
Pushed.

**Pull request closed without merging**